### DISCUSSION TITLE
Button layout when displaying only a button link gets "lost"

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1683] [Proposals]
### Goal

To be able to have a left aligned button link when used inside a button layout
<img width="304" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/183221f9-e40e-48a1-a8cd-d4b960b5fd1f">

### Context

<img width="315" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/3bf03d07-386f-4aa8-a9a2-8cd1064d95af">
Now, due to the button layout definition, a button link looks like it's floating, even if the left aligment prop is on.

### Impact

CMS

### Discussion Questions

¿Could we make the button link, when alone and left aligned, not be reescaled to occupy the 50% of space?

**Comment by aweell** [comment]
We'll add this to the update of: https://github.com/Telefonica/mistica-design/issues/1674
